content/docs/tutorials/getting-started-with-cert-manager-on-azure-kubernetes-service-using-lets-encrypt-for-ssl/README.md

```diff
@@ -0,0 +1,806 @@
+
+```bash
+az network dns zone show --name $DOMAIN_NAME --output yaml
+```
+
+You can check that the NS records have been updated using `dig` to "trace" the hierarchy of NS records,
+rather than using your local DNS resolver:
+
+```bash
+dig $DOMAIN_NAME ns +trace +nodnssec
+```
+
+> ⏲ It **may** take more than 1 hour for the NS records to be updated in the parent zone,
+> and it may take some time for the old NS records to be replaced in the caches of DNS resolver servers,
+> if you looked up the DNS name before updating the NS records.
+>
+> 📖 Read [How do I Update My DNS Records?](https://docs.gandi.net/en/domain_names/common_operations/dns_records.html) in the `Gandi.net` docs,
+> or seek the equivalent documentation for your own domain name registrar.
+
+## Create a Kubernetes cluster
+
+To get started, let's create a Kubernetes cluster in Microsoft Azure.
+You will need to pick a name for your cluster.
+Here, we will go with "test-cluster-1".
+Save it in an environment variable:
+
+```bash
+export CLUSTER=test-cluster-1
+```
+
+Now, create the cluster using the following command:
+
+```bash
+az aks create \
+    --name ${CLUSTER} \
+    --node-count 1 \
+    --node-vm-size "Standard_B2s" \
+    --load-balancer-sku basic
+```
+
+Update your `kubectl` config file with the credentials for your new cluster:
+
+```bash
+az aks get-credentials --admin --name "$CLUSTER"
+```
+
+Now check that you can connect to the cluster:
+
+```bash
+kubectl get nodes -o wide
+```
+
+> ⏲ It will take 4-5 minutes to create the cluster.
+>
+> 💵 To minimize your cloud bill, this command creates a 1-node cluster using a
+> low cost virtual machine and load balancer.
+>
+> ⚠️ This cluster is only suitable for learning purposes it is not suitable for production use.
+>
+> 📖 Read [Run Kubernetes in Azure the Cheap Way](https://trstringer.com/cheap-kubernetes-in-azure/) for more cost saving tips.
+>
+
+## Install cert-manager
+
+Now you can install and configure cert-manager.
+
+Install cert-manager using `helm` as follows:
+
+```bash
+helm repo add jetstack https://charts.jetstack.io
+helm repo update
+helm upgrade cert-manager jetstack/cert-manager \
+    --install \
+    --create-namespace \
+    --wait \
+    --namespace cert-manager \
+    --set installCRDs=true
+```
+
+This will create three Deployments and some Services and Pods in a new namespace called `cert-manager`.
+It also installs various cluster scoped supporting resources such as RBAC roles and Custom Resource Definitions.
+
+You can view some of the resources that have been installed as follows:
+
+```bash
+kubectl -n cert-manager get all
+```
+
+And you can explore the Custom Resource Definitions (cert-manager's API) using `kubectl explain`, as follows:
+
+```bash
+kubectl explain Certificate
+kubectl explain CertificateRequest
+kubectl explain Issuer
+```
+
+> 📖 Read about [other ways to install cert-manager](../../installation).
+>
+> 📖 Read more about [Certificates and Issuers](../../concepts).
```

Comment:
suggestion: again, https://github.com/cert-manager/website/#task-linking-to-other-documentation-pages

Same applies to any other internal documentation links on here; I've not got time to check them all ATM unfortunatley!